index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Page Title</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <script src='main.js'></script>
+</head>
+
+<body>
+  <div class="main-container">
```

Comment:
- [ ] Great job with the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp), just another little thing. It'd be nice if you replaced the `<div>` tag with a `<header>` tag right? What do you think!